pkg/network/tracer/tracer_classification_test.go

```diff
@@ -90,43 +90,15 @@ const (
 	rawTrafficPort = "9093"
 )
 
-func testProtocolClassification(t *testing.T, tr *Tracer, clientHost, targetHost, serverHost string) {
+func testProtocolClassificationCrossOS(t *testing.T, tr *Tracer, clientHost, targetHost, serverHost string) {
```

Comment:
will this test func be ever executed? it is called from non linux env? looks like the called has a `linux_bpf` build tag

Reply:
yes, it will be executed on linux

if you don't have any specific tag (we don't have in `tracer_classification_test.go`) it can be used everywhere